Project.toml

```
@@ -3,7 +3,7 @@ uuid = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "A lightweight interface for common MCMC methods."
-version = "5.1.0"
+version = "5.1.1"
```

Following semver, I think we should change this to
```suggestion
version = "5.2.0"
```
since the PR is not a bugfix but a new feature.